app/Http/Controllers/StoreController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class StoreController extends Controller
+{
+  function store(Request $request)
+  {
+
+      $name = $request->input('name');
```

Comment:
Hello Omneya,,

your code does the job but i suggest a little enhancement

look at this
```php
$request->input('name');
```

better syntax
```php
$request->name;
```

or
```php
$request['name'];
```